pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/proj/main/SalesApplication.java

```diff
@@ -0,0 +1,88 @@
+package com.subho.wipro.pjp.tm08.proj.main;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+import com.subho.wipro.pjp.tm08.proj.bean.Product;
+import com.subho.wipro.pjp.tm08.proj.bean.Sales;
+import com.subho.wipro.pjp.tm08.proj.service.Administrator;
+import com.subho.wipro.pjp.tm08.proj.util.DBUtil;
+
+public class SalesApplication {
+
+	public static void main(String[] args) throws ParseException {
+		Scanner sc = new Scanner(System.in);
+		
+		Administrator admin = new Administrator();
+		
+		int choice = 0;
+		
+		do {
+			System.out.println("1. Insert Stock");
+			System.out.println("2. Delete Stock");
+			System.out.println("3. Insert Sales");
+			System.out.println("4. View Sales Report");
+			System.out.print("Enter your Choice: ");
+			choice = sc.nextInt();
+			
+			switch (choice) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A switch statement does not contain a break](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)